### PR TITLE
Feature/tree builder spec

### PR DESCRIPTION
**Pull request by Djeg**

### CODE REVIEW

File: spec/Knp/RadBundle/AppBundle/ConfigurationFactorySpec.php [hunk 1]
```diff
@@ -16,4 +16,5 @@ function it_should_create_configuration($bundle, $container)
         $config->shouldBeAnInstanceOf('Knp\RadBundle\AppBundle\Configuration');
         $config->getBundle()->shouldReturn($bundle);
     }
+
```
Comment (docteurklein) [inline spec/Knp/RadBundle/AppBundle/ConfigurationFactorySpec.php]: why this? not really important btw

File: spec/Knp/RadBundle/AppBundle/ConfigurationSpec.php [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
 
-class Configuration extends ObjectBehavior
+class ConfigurationSpec extends ObjectBehavior
 {
     /**
      * @param Knp\RadBundle\AppBundle\ConfigurableBundleInterface $bundle
-     * @param Knp\RadBundle\AppBundle\TreeBuilderFactory          $treeBuilderFactory
+     * @param Knp\RadBundle\AppBundle\TreeBuilderFactory $treeBuilderFactory
```
Comment (docteurklein) [inline spec/Knp/RadBundle/AppBundle/ConfigurationSpec.php]: you could have kept names alignment

File: spec/Knp/RadBundle/AppBundle/ConfigurationSpec.php [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
      */
     function let($bundle, $treeBuilderFactory)
     {
         $this->beConstructedWith($bundle, $treeBuilderFactory);
     }
-
+
```
Comment (docteurklein) [inline spec/Knp/RadBundle/AppBundle/ConfigurationSpec.php:17]: extra useless spaces

**Comment by docteurklein** [comment]
that was fast :) thanks!

**Comment by Djeg** [comment]
sorry for the typos. My finger seems to no like my .vimrc + keyboards :-( !!

**Comment by docteurklein** [comment]
:) no prob

**Comment by gquemener** [comment]
vim :heart:

**Pull request merged**